### DISCUSSION TITLE
Oracle support

### DISCUSSION THREAD

**Discussion by Wintermute79** [MoonStorm/FastCrud#165] [Ideas]
Do you intend to support Oracle?
Your benchmarks are awesome but at the moment only Dapper-Extensions seems to have Oracle support. See https://github.com/tmsmith/Dapper-Extensions/blob/master/DapperExtensions/Sql/OracleDialect.cs

**Comment by MoonStorm** [comment]
Oracle support is up for grabs for anyone knowledgeable enough. I expect it to be a pain especially for identity columns... ahem, columns having values generated through sequence objects 😠
